contracts/oracles/OraclePoolToken.sol

```diff
@@ -96,7 +96,8 @@ contract OraclePoolToken is IOracleUsd {
             uint a = aPool.mul(eAvg);
             uint b = a.mul(9).div(Q112);
             uint c = ePool.mul(3988000);
-            uint sqRoot = sqrt(a.div(Q112).mul(b.add(c)));
+            //uint sqRoot = sqrt(a.div(Q112).mul(b.add(c)));
+            uint sqRoot = b.add(c).mul(sqrt(a)).div(sqrt(Q112));
```

Comment:
Currently there are conflicts after removal of SafeMath with update to Solidity 0.8.15, this PR needs to be updated plus we need to make sure the tests pass after this change